editoast/src/views/projection.rs

```diff
@@ -394,6 +428,201 @@ pub async fn compute_projected_train_paths(
     Ok(project_path_result)
 }
 
+fn linear_interpolate(pos: &[u64], op_begin: u64, op_end: u64) -> Vec<u64> {
```

Comment:
This function signature is weird.

`pos` is provided as a list
`op`, begin and end are given as flat input
The output is provided as list.

Using a list is not ideal when we know its arity.

```suggestion
fn linear_interpolate(a_x: u64, a_y: u64, b_x: u64, b_y: u64) -> (u64, u64) {
```

Here we have something consistent and well-defined.

Reply:
> [!NOTE]
> We already have a function for linear interpolation. See `views::projections::interpolate`.